src/plot_settings.py

```diff
@@ -33,45 +33,50 @@ class PlotSettingsWindow(Adw.PreferencesWindow):
     max_top = Gtk.Template.Child()
     no_data_message = Gtk.Template.Child()
 
+    @property
+    def plot_settings(self):
+        return self.props.application.plot_settings
```

Comment:
Any particular reason to define the `plot_settings` as property of the class like this, instead of retrieving it as an attribute using `self.props.application.plot_settings`?

Reply:
Yeah I was just wondering if it perhaps isn't cleaner to just call `self.plot_settings = self.props.application.plot_setting
` in the init instead of having it as a property using the decorator, in particular since it's not really a fixed/read-only property of the class itself but rather an external attribute of the application that we're modifying. 
On the other hand, in the end, the property is kinda still static here, in the sense that it's still the same reference to the same attribute. That is still read-only, so in that argument using @property maybe does make sense. But my initial instinct would be to just go for an attribute directly. 

Either would be better than calling `plot_settings = self.props.application.plot_settings` each time by the way, it's reused throughout the class often enough that it should be defined as instance of the class.

Reply:
It was just initial instinct to do it this way, but moving it to init works just was well, I don't believe there is a measurable difference in memory consumtion/cpu usage either way